src/main/java/com/agilethought/atsceapi/validator/ValidationUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.agilethought.atsceapi.validator;
+
+public class ValidationUtils {
+	public static boolean isStringLowerCase(String str) {
+		// convert String to char array
+		char[] charArray = str.toCharArray();
+		for (int i = 0; i < charArray.length; i++) {
+			// if the character is a letter
+			if (Character.isLetter(charArray[i])) {
+				// if any character is not in lower case, return false
+				if (!Character.isLowerCase(charArray[i]))
+					return false;
+			}
+		}
+		return true;
+	}
+
+	public static boolean isValid(String email) {
```

Comment:
The name of this method is ambiguos, maybe we can change this name as isValidEmail